framework/doc/content/source/mfem/auxkernels/MFEMGradAux.md

```diff
@@ -5,7 +5,7 @@
 ## Overview
 
 AuxKernel for calculating the gradient of a scalar $H^1$ conforming source variable and storing it in
-a scalar elemental result variable defined on an $H(\mathrm{curl})$ conforming Nédélec finite element space.
+a result variable defined on an $H(\mathrm{curl})$ conforming Nédélec finite element space.
```

Comment:
why do we map to a nedelec space instead of a vector L2?

Reply:
What do you do in 1D?

Reply:
I understand now that while obviously you do not do a good job of representing the finite element solution gradient $\nabla u_h$ by projecting into the Nedelec space, you do a *better* job of representing the true solution gradient by projecting into the Nedelec space 👍 

> What do you do in 1D?

Oh I see that MFEM supports Nedelec on segments. Nice